src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PhysicalDetailService.kt

```diff
@@ -138,6 +138,63 @@ class PhysicalDetailService(
 
       minShoeSize?.let { detailQuery.filter(rangeQuery("shoeSize").gte(it)) }
       maxShoeSize?.let { detailQuery.filter(rangeQuery("shoeSize").lte(it)) }
+
+      tattoos?.forEach { tattoo ->
+        detailQuery.filter(
+          boolQuery().mustAll(
+            buildList {
+              tattoo.bodyPart?.let {
+                add(QueryBuilders.matchQuery("tattoos.bodyPart", tattoo.bodyPart))
+              }
+              tattoo.comment?.let {
+                add(QueryBuilders.matchQuery("tattoos.comment", tattoo.comment))
+              }
+            }
+          )
+        )
+      }
+      scars?.forEach { scar ->
+        detailQuery.filter(
```

Comment:
bit of duplication here with the queries - will sort out in the next PR as need to work out what should be `must` and what should be `should`